configs/pontoon.toml

```diff
@@ -284,3 +284,6 @@ locales = [
 [[paths]]
     reference = "en/products/vpn/shared.ftl"
     l10n = "{locale}/products/vpn/shared.ftl"
+[[paths]]
+    reference = "en/products/vpn/more/*.ftl"
```

Comment:
Paths not in the proper sorting order. There are  a few paths that are  out of place. Let's fix them.